eq-author/src/App/keyboardShortcuts/KeyboardShortcutsPage.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import Header from "components/EditorLayout/Header";
+
+import styled from "styled-components";
+
+import { colors } from "constants/theme";
+
+import {
+  Layout,
+  SharePageTitle,
+  Description,
+} from "../sharing/styles/SharePageContent";
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`;
+
+const StyledLayout = styled.div`
+  width: 100%;
+  height: 100%;
+`;
+
+const TableOuter = styled.table`
+  width: 100%;
+  border: 1px solid ${colors.grey35};
+  padding: 0;
+`;
+
+const TableRow = styled.tr`
+  border-bottom: 1px solid ${colors.grey35};
+`;
+
+const TableHeader = styled.td`
+  width: 50%;
+  padding: 1em;
+  text-align: left;
+  font-size: 1.25em;
+`;
+
+const TableData = styled.td`
+  padding: 1em;
+  text-align: left;
+  font-size: 1.25em;
+`;
+
+const TableDataB = styled.td`
```

Comment:
```suggestion
const TableDataDarkGrey = styled.td`
```
Suggested name change